tests/python/gaia-ui-tests/gaiatest/apps/contacts/regions/merge.py

```diff
@@ -0,0 +1,27 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+from marionette_driver import expected, By, Wait
+
+from gaiatest.apps.base import PageRegion
+
+
+class Merge(PageRegion):
+
+    _iframe_locator = (By.CSS_SELECTOR, '.popupWindow.active iframe[data-url*="matching"]')
+    _merge_button = (By.ID, 'merge-action')
+
+    def __init__(self, marionette):
+        marionette.switch_to_frame()
+        
+        root = Wait(marionette).until(
+            expected.element_present(*self._iframe_locator))        
+        Wait(marionette).until(expected.element_displayed(root))
+        marionette.switch_to_frame(root)
+
```

Comment:
I investigated the current test code a bit more, and it turns out it is slow enough to not fail. However, if one day the device becomes too slow, our test would fail for an unknown reason. Hence, here's the wait I propose to add:
```py
_body_locator = (By.TAG_NAME, 'body') 

def __init__(self, marionette):
    # ...
    self.marionette.switch_to_frame(root)
    body = self.marionette.find_element(*self._body_locator)
    Wait(self.marionette).until(lambda m: body.rect['y'] == 0)
```